components/server/src/user/user-deletion-service.ts

```diff
@@ -58,7 +57,12 @@ export class UserDeletionService {
         }
 
         // Stop all workspaces
-        await this.stopWorkspaces(user);
+        await this.workspaceStarter.stopRunningWorkspacesForUser(
+            {},
+            user.id,
+            "user deleted",
+            StopWorkspacePolicy.IMMEDIATELY,
```

Comment:
This used to be policy "NORMALLY" but killing all workspaces immediately makes more sense in case of user deletion.